### DISCUSSION TITLE
TE3  &datamodel

### DISCUSSION THREAD

**Discussion by koltachin** [TabularEditor/TabularEditor3#347] [Ideas]
I was wondering since TE3 works with datamodel does this mean we can use TE3 in excel power pivot datamodel?
kind regards

roger

**Comment by otykier** [comment]
No, unfortunately, since Excel uses an older version of Analysis Services internally. For TE2/TE3 to work, the compatibility level must be 1200 or higher, but Excel uses 1100/1103.

**Comment by koltachin** [comment]
wouldn't it be nice to have the functionality of excel's F9 shortcut  ,  push F9 on an selected variable or measure to see the result in a scalar value or table !

desperatly waiting for the dax debugger 

kind regards
Roger

**Comment by koltachin** [comment]
tested the dax debugger but the application freezed and had to close it by CTRL+ALT +DEL
attached the last screen
![dax debugger](https://user-images.githubusercontent.com/46677876/151424710-149e2633-600b-4ab3-88c6-3ef219a6dcec.JPG)
L

**Reply by otykier** [reply]
What was the last thing you did before the application froze? Other users have reported that [a freeze sometimes occurs when dragging windows around](#354).

**Comment by koltachin** [comment]
Daniel

 

If I remember correctly : -->When I dragged the call tree  window aside

 

kind regards

 

roger

**Reply by otykier** [reply]
Ok - in that case, I think it is the same issue as the other user reported. If you can provide step-by-step instructions on how to reproduce the application freeze, please let us know (please add your comments to #354).
